com.reprezen.swagedit/src/com/reprezen/swagedit/editor/SwaggerSourceViewerConfiguration.java

```diff
@@ -57,7 +57,7 @@ public IContentAssistant getContentAssistant(ISourceViewer sourceViewer) {
         ca.setContentAssistProcessor(processor, IDocument.DEFAULT_CONTENT_TYPE);
         ca.setInformationControlCreator(getInformationControlCreator(sourceViewer));
 
-        ca.enableAutoInsert(true);
+        ca.enableAutoInsert(false);
```

Comment:
I think this looks right, but it's so darned complicated! I'll just have to test it and see... :)